### PR TITLE
Add busy version to settings page

### LINKED ISSUE #1811: Version number in settings
Showing Busy version number in settings page could be useful for debugging and referencing.

### PR DESCRIPTION
**Pull request by kirkins**
Fixes #1811 by adding the suggested version number to the settings page.

### CODE REVIEW

**Comment by kirkins** [comment]
Added some padding to the version, not sure what it should look like.

File: src/client/settings/Settings.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ import requiresLogin from '../auth/requiresLogin';
 import LANGUAGES from '../translations/languages';
 import { getLanguageText } from '../translations';
 import './Settings.less';
+import PackageJson from '../../../package.json';
```
Comment (Sekhmet) [inline src/client/settings/Settings.js]: I would just call this variable `package`.

**Comment by Sekhmet** [comment]
Thanks. You should add default translation to default.json as well:
https://github.com/busyorg/busy/blob/master/docs/CONTRIBUTING.md#adding-new-translations

This guide doesn't cover adding translations with variables yet, but you should create variable:
`id: "version"`
`defaultMessage: "Version: {version}".`

Otherwise we would have to translate each version to all languages.

See values prop for FormattedMessage: https://github.com/yahoo/react-intl/wiki/Components#formattedmessage

**Comment by Sekhmet** [comment]
@bonustrack what text should we use?

**Comment by bonustrack** [comment]
"Version: {version}" like this without "Busy" is fine.

**Comment by kirkins** [comment]
Did I miss anything for adding the ID?

File: src/client/settings/Settings.js (hunk 1 above)
Comment (kirkins) [reply src/client/settings/Settings.js]: Seems the word is reserved. I'll make the p lowercase.

**Comment by Sekhmet** [comment]
Formatted message should look like this
```js
<FormattedMessage id="version" defaultMessage="Version: {version}" values={{ version: package.version }} />
```

Can you rename translation ID to just `version`?

**Pull request merged**